src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -611,7 +611,7 @@ export function getAlterationEnrichmentColumns(
         enrichedGroupColum.tooltip = (
             <table>
                 <tr>
-                    <td>Log ratio > 0</td>
+                    <td>Log ratio {'>'} 0</td>
```

Comment:
officially, JSX does not allow >, <, {, } as plaintext characters, and the new version of typescript catches that